### PR TITLE
PICKS-223 Facebook Auth: Fixed issue with users logging in with a Fac…

### PR DESCRIPTION
**Pull request by the-freshlord**
…ebook account with no email.

### CODE REVIEW

File: common/models/picks-user.js [hunk 1]
```diff
@@ -94,12 +140,7 @@ module.exports = function(PicksUser) {
...
-      var Group = PicksUser.app.models.Group;
-      var Score = PicksUser.app.models.Score;
-      var Season = PicksUser.app.models.Season;
-      var isNewUser = false;
-      var globalGroup;
-      let userResponse, accessTokenId, newUser, newScore, facebookAccessToken;
+      let userResponse, facebookAccessToken, facebookId;
       // Exchange oauth token for access token
       Facebook.accessToken(appId, appSecret, redirectUrl, oauthToken)
       .then(function(response) {
```
Comment (sundayguru) [inline common/models/picks-user.js:146]: Remove this since it's not used

**Review by sundayguru — APPROVED** [review]

File: common/models/picks-user.js (hunk 1 above)
Comment (the-freshlord) [reply common/models/picks-user.js:146]: Which part exactly?

Comment (mightys) [reply common/models/picks-user.js:146]: line 146; the whole `then` chain

Comment (the-freshlord) [reply common/models/picks-user.js:146]: No its used

Comment (the-freshlord) [reply common/models/picks-user.js:146]: The accessToken method returns a promise thats handled by the then chain.

Comment (mightys) [reply common/models/picks-user.js:146]: You're right, I didn't see the collapsed snippet

**Pull request merged**